livemaps-api/APIFunctions/Config.cs

```diff
@@ -1,23 +1,20 @@
 using System;
-using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
 using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
-using System.ComponentModel;
-using System.Text;
-using ssir.api.Services;
-using System.Collections.Generic;
 using ssir.api.Models;
-using Microsoft.WindowsAzure.Storage.Blob;
-using System.Linq.Expressions;
-using System.Linq;
-using System.Net.Http;
 using ssir.api.Models.Atlas;
-using System.Net;
+using ssir.api.Services;
 
 namespace ssir.api
```

Comment:
This namespace does not follow C# norms for casing.  As an official sample, we should be.   For example, this should be `Ssir.Api.Services`.    

Leaving one comment here, but it applies throughout the sample.